BeVolt/BSP/Simulator/DataGeneration/simulate.py

```diff
@@ -53,16 +62,21 @@ def configure():
     # Get configuration settings
     global state, mode
     print("Welcome to the BPS Simulator")
-    print("Would you like to simulate 'charging' or 'discharging'?")
+    print("Would you like to simulate 'charging', 'discharging', or 'PLL'?")
     state = input()
-    while state != 'charging' and state != 'discharging':
-        print("That is not a valid option. Please enter 'charging' or 'discharging': ")
+    while state != 'charging' and state != 'discharging' and state != 'PLL':
+        print("That is not a valid option. Please enter 'charging', 'discharging', or 'PLL': ")
```

Comment:
I saw on the CAN simulator pull request Rishi mentioned adding the any configuration stuff for whenever the user pressed CTRL+C. Specifically, handling it in the configure function